test/definitions/procedures/packages/update_all_confirmation_test.rb

```diff
@@ -24,11 +24,11 @@ def skip_mock_package_manager
       To upgrade to the next Foreman version use 'foreman-maintain upgrade'.
       Do you want to proceed with update of everything regardless of
       the recommendations?
+      , [y(yes), q(quit)]
```

Comment:
What's the weird comma here?

Reply:
That was the original point of me re-working the tests, the output is weird. And I first wanted to create a test that can show that and then follow up with a commit that fixes it.

Reply:
That's always useful information to include in a PR description.